### PR TITLE
Fix/readme

### PR DESCRIPTION
**Pull request by flovntp**
Please have a look on this page to test README file 
https://github.com/flovntp/platformsh-symfony-template/tree/fix/readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,761 @@
...
+</a>&nbsp&nbsp
+
+[comment]: <> (<a href="https://github.com/symfonycorp/platformsh-symfony-template/blob/master/LICENSE">)
+
+[comment]: <> (<img src="https://img.shields.io/static/v1?label=License&message=MIT&style=for-the-badge&labelColor=f4f2f3&color=ffd9d9" alt="License" />)
+
+[comment]: <> (</a>&nbsp&nbsp)
+
+[comment]: <> (<br /><br />)
+<a href="https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/symfonycorp/platformsh-symfony-template-metadata/main/template-metadata-sf6.0-php8.1-webapp.yaml&utm_content=symfonycorp&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform">
```
Comment (chadwcarlson) [inline README.md]: I think these needs to be in it's own `align="center"` `p` it looks like

File: README.md [hunk 2]
```diff
@@ -1,8 +1,761 @@
...
+## About
+
+<p>
+   We have just created a new <a href="https://symfony.com/doc" target="_blank">Symfony</a> project for you.
+   The generated code has been optimized for a typical web application hosted on <a href="https://platform.sh/">Platform.sh</a>.
+</p>
+
+### Features
+
+<p>
```
Comment (chadwcarlson) [inline README.md:62]: This is a major improvement on our *Features* section. Nice work.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,761 @@
...
+
+#### Other deployment options
+
+For all of the other options below, clone this repository first:
+
+```bash
+git clone https://github.com/symfonycorp/platformsh-symfony-template.git
+```
+
+If you're trying to deploy from GitHub, you can generate a copy of this repository first in your own namespace by clicking the [Use this template](https://github.com/symfonycorp/platformsh-symfony-template/generate) button at the top of this page.
```
Comment (chadwcarlson) [inline README.md:127]: This is another @fabpot todo - marking this repository as a **Template repository** https://github.com/symfonycorp/platformsh-symfony-template/settings

File: README.md [hunk 4]
```diff
@@ -1,8 +1,761 @@
...
+
+The `platform mount:upload` command provides a straightforward way to upload an entire directory to your site at once to a `mount` defined in a `.platform.app.yaml` file.
+Under the hood, it uses an SSH tunnel and rsync, so it is as efficient as possible.
+(There is also a `platform mount:download` command you can use to download files later.)
+Run the following from your local Git repository root (modifying the `--source` path if needed and setting `BRANCH_NAME` to the branch you are using).
+
+A few examples are listed below, but repeat for all directories that contain data you would like to migrate.
+
+```bash
+$ platform mount:upload -e main --mount web/sites/default/files --source ./web/sites/default/files
```
Comment (chadwcarlson) [inline README.md:583]: Looks like `var` may be the only dir that needs uploading? https://github.com/symfonycorp/platformsh-symfony-template/blob/bb433871d2dda2d29b815776a0af43f5ffaef7ea/.platform.app.yaml#L36

File: README.md [hunk 5]
```diff
@@ -1,8 +1,761 @@
...
+
+### Contact
+
+This template is maintained by the Platform.sh Developer Relations team, and they will be notified of all issues and pull requests you open here.
+
+- **Community:** Share your question with the community, or see if it's already been asked on our [Community site](https://community.platform.sh).
+- **Slack:** If you haven't done so already, you can join Platform.sh's [public Slack](https://chat.platform.sh/) channels and ping the `@devrel_team` with any questions.
+
+
+### About Platform.sh
```
Comment (chadwcarlson) [inline README.md:682]: @flovntp I'd be curious if you have feedback on what should be written here. Is there too much? Too little? Something missing?

File: README.md (hunk 4 above)
Comment (flovntp) [reply README.md:583]: yes

File: README.md (hunk 5 above)
Comment (flovntp) [reply README.md:682]: @chadwcarlson I think it's enough to understand what PSH is and offers. 
Do you see something missing on your POV?

File: README.md [hunk 6]
```diff
@@ -1,8 +1,755 @@
...
+    <img src="https://platform.sh/images/deploy/deploy-button-lg-blue.svg" alt="Deploy on Platform.sh" width="175px">
 </a>
 </p>
+<br/>
+
+
+You can also quickly recreate this project locally with the following command:
+
+```bash
+#TODO @FabienPotencier add this repo to Packagist
```
Comment (flovntp) [inline README.md]: Hi @fabpot could you please add this repo to packagist to let user create project using the console?

**Comment by flovntp** [comment]
@fabpot 
Could you please push this repo to packagist? 
or do we remove the `composer create-project ...` from the readme?

**Pull request closed without merging**